src/FEValues/face_integrals.jl

```diff
@@ -1,113 +1,134 @@
+"""
+    reference_face_to_face(point::Vec{N, T}, cell_T::Type{<:AbstractRefShape}, face::Int) 
+
+Transform point from face's reference (N-1)D coordinates to ND coordinates on the cell's face.
+"""
+reference_face_to_face
```

Comment:
I like the idea! Can we come up with a better name for the routine maybe?

Reply:
Thanks!
Will think about a better one.